src/components/Spinner/index.js

```diff
@@ -1,16 +1,21 @@
-import { h } from 'preact'
+import { h, Component } from 'preact'
 import { compose } from '~utils/func'
 import { localised } from '../../locales'
 import style from './style.scss'
 
-const Spinner = ({ translate }) => {
-  return (
+class Spinner extends Component {
+  componentDidUpdate() {
+    this.container && this.container.focus()
```

Comment:
IMHO it's worth adding an explanation here
```suggestion
    /**
     * Until iOS 14, the loading screen does not get announced by VoiceOver on iOS
     * This manual focus() trigger replaces `autoFocus` prop on the main `div`
     * to make sure it works properly on every platform.
     */
    this.container && this.container.focus()
```

Reply:
It's broken on iOS14 too. I will add comments as inline documentation

Reply:
I've added comments, let me know your thoughts